src/__csp-nonce.ts

```diff
@@ -19,6 +19,24 @@ type Params = {
 const params = inputs as Params;
 
 const handler = async (request: Request, context: Context) => {
+  const isGET = request.method.toUpperCase() === "GET";
```

Comment:
I think this normalisation should be handled automatically for us.

```suggestion
  const isGET = request.method === "GET";
```